README.md

```diff
@@ -22,27 +21,25 @@ This is the backend for E-Commerce-Ninjas, written in Node.js with TypeScript.
 
 [https://github.com/atlp-rwanda/e-commerce-ninjas-bn](https://github.com/atlp-rwanda/e-commerce-ninjas-bn)
 
-
 ## COMPLETED FEATURES
 
 - Welcome Endpoint
 - Register Endpoint
 - Verification Email Endpoint
 - Resend verification Endpoint
 - Login Endpoint
+- Logout Endpoint
 
 ## TABLE OF API ENDPOINTS SPECIFICATION AND DESCRIPTION
 
-
-| No | VERBS | ENDPOINTS                    | STATUS      | ACCESS | DESCRIPTION               |
-|----|-------|------------------------------|-------------|--------|---------------------------|
-| 1  | GET   | /                            | 200 OK      | public | Show welcome message      |
-| 2  | POST  | /api/auth/register           | 201 CREATED | public | create user account       |
-| 3  | GET   | /api/auth/verify-email/:token| 200 OK      | public | Verifying email           |
-| 4  | POST  | /api/auth/send-verify-email  | 200 OK      | public | Resend verification email |
-| 4  | POST  | /api/auth/login              | 200 OK      | public | Login with Email and Password |
-
-
+| No  | VERBS | ENDPOINTS                     | STATUS      | ACCESS | DESCRIPTION                   |
+| --- | ----- | ----------------------------- | ----------- | ------ | ----------------------------- |
+| 1   | GET   | /                             | 200 OK      | public | Show welcome message          |
+| 2   | POST  | /api/auth/register            | 201 CREATED | public | create user account           |
+| 3   | GET   | /api/auth/verify-email/:token | 200 OK      | public | Verifying email               |
+| 4   | POST  | /api/auth/send-verify-email   | 200 OK      | public | Resend verification email     |
+| 5   | POST  | /api/auth/login               | 200 OK      | public | Login with Email and Password |
+| 6   | POST  | /api/auth/logout              | 200 OK      | public | Logout user                   |
```

Comment:
The access is not `Public`, should be `Private`.  Because you can't access Logout without Logged-In.